packages/core/src/components/CheckBox/index.tsx

```diff
@@ -20,6 +20,7 @@ export default class CheckBox extends React.Component<Props, State> {
     button: false,
     checked: false,
     children: null,
+    neutral: false,
```

Comment:
Is this that state with the dash in it? It already has a name, it's called `indeterminate`, so please use that.

Reply:
I don't see this prop on `Checkbox` and got 0 matches for `indeterminate` in the repo

Reply:
But yeah, it's this state:
<img width="38" alt="Screen Shot 2019-05-06 at 11 43 04 AM" src="https://user-images.githubusercontent.com/8676510/57247227-21209e80-6ff4-11e9-9574-163667adc96a.png">

Reply:
Yeah, let's use the official name instead of neutral.